drafts/concise_guide.ipynb

```diff
@@ -92,7 +92,66 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When creating a response, FastHTML will automatically render tags using the to_xml function. "
+    "This example demonstrates key aspects of how FastHTML components handle attributes:\n",
+    "\n",
+    "```python\n",
+    "Label(\n",
+    "    \"Choose an option\", \n",
+    "    Select(\n",
+    "        Option(\"one\", value=\"1\", selected=True),  # Boolean True renders just the attribute name\n",
+    "        Option(\"two\", value=\"2\", selected=False), # Boolean False omits the attribute entirely\n",
+    "        Option(\"three\", value=3),                 # Non-string values are converted to strings\n",
+    "        cls=\"selector\",                           # 'cls' becomes 'class' in HTML (Python reserved word)\n",
```

Comment:
We don't normally use `_id`, but just `id`, and we want to only show one way to do things. You can mention the underscore strip behavior in the markdown below briefly instead.